crates/containerd-shim-wasmtime/src/instance.rs

```diff
@@ -22,6 +23,18 @@ use crate::http_proxy::serve_conn;
 
 pub type WasmtimeInstance = Instance<WasmtimeEngine<DefaultConfig>>;
 
+static PRECOMPILER: LazyLock<wasmtime::Engine> = LazyLock::new(|| {
+    let mut config = wasmtime::Config::new();
+
+    // Disable Wasmtime parallel compilation for the tests
+    // see https://github.com/containerd/runwasi/pull/405#issuecomment-1928468714 for details
+    config.parallel_compilation(!cfg!(test));
```

Comment:
Does this not run the risk of configurations becoming misaligned with how the runner engine is configured?